### ISSUE TITLE
Proposal: Median option

### ISSUE THREAD

**Issue by yuonoda** [fzipp/gocyclo#54]
Hey, thank you for your providing nice tool ! 
I would like  you to add new option to show median of cyclomatic complexity.

I often meet occasions that a distribution of  cyclomatic complexity in my code is highly skewed.
For example, a complexity of almost all functions in my repo is around 1.
But, a little functions marks 10 to 100 complexity.
In such case, average complexity tend to be higher than median or mode, so cannot capture whole trend.

I propose to inplement `median` option to this command like this,

```
    -med, -med-short      show the median complexity over all functions;
                          the short option prints the value without a label
```

This should be helpful for those who meet same issue with me.

Thanks,

**Comment by fzipp** [comment]
I agree that the median is useful. Would you like to implement it and create a pull request?

**Comment by fzipp** [comment]
Someone created a pull request for a percentile option: #55
This would cover the median as the 50th percentile. You may want to provide your input there as well.
